pkg/apic/consumerinstance.go

```diff
@@ -44,6 +44,11 @@ func (c *ServiceClient) buildConsumerInstanceSpec(serviceBody *ServiceBody, doc
 			if _, found := teamMap[serviceBody.TeamName]; found {
 				owningTeam = serviceBody.TeamName
 			} else {
+				if owningTeam == "" {
+					owningTeam = "the default team"
```

Comment:
Maybe I'm missing something.  But why not set this as the default?

Reply:
I guess I can

Reply:
Ok, i didn't scroll high enough to see it.